.gitignore

```diff
@@ -3,3 +3,14 @@
 
```

Comment:
Can we stick this in here too please:

# Exclude all VSCode configuration files.
**.vscode/